src/models/TalkMapper.php

```diff
@@ -772,4 +772,16 @@ public function delete($talk_id)
 
         return true;
     }
+
+    public function getSpeakerFromTalk($talk_id, $display_name)
+    {
+        $speaker_sql  = 'select ts.* from talk_speaker ts '
+            . 'where ts.talk_id = :talk_id and ts.speaker_name = :display_name';
+        $speaker_stmt = $this->_db->prepare($speaker_sql);
+        $speaker_stmt->execute(array("talk_id" => $talk_id, "display_name" => $display_name));
+        $speakers = $speaker_stmt->fetchAll(PDO::FETCH_ASSOC);
+
+        return count($speakers) > 0 ? $speakers[0] : false;
```

Comment:
Wouldn't it be easier to throw an exception when no speaker was found? That way you either receive an Exception from this method or an array containing user-values. So you wouldn't need to check the return-value for a boolean ```false``` every time. That might then refactor https://github.com/joindin/joindin-api/pull/352/files#diff-e27b217374c8436f4b830c74ed46cbbaR550